### ISSUE TITLE
Can no longer add scripts to docker-entrypoint-init.d

### ISSUE THREAD

**Issue by fatjester** [bitnami/containers#2698]
### Name and Version

bitnami/wordpress-nginx:6.0.1-debian-11-r12

### What steps will reproduce the bug?

using provided docker-compose.yml

adding volume mapping to `docker-entrypoint-init.d`:
```yaml
 - './wp-scripts:/docker-entrypoint-init.d'
```
run docker-compose up

### What is the expected behavior?

successful install run of a custom script after the WordPress core install

### What do you see instead?
```
mkdir: cannot create directory '/bitnami/wordpress-nginx': Permission denied
```
### Additional information

This looks to be related to this recent change:
https://github.com/bitnami/containers/commit/4b1a11c981728201ee82157c60f91bae20907438#diff-44ce69ef8cd01ce686cb579f79cf7d5bdc38531d1a8f630c17935ad4b59e48a6

**Comment by fmulero** [comment]
Thanks @fatjester for reporting this issue.`wordpress-intel` asset is affected by the same issue

**Comment by dgomezleon** [comment]
Hi @fatjester ,

You are right. Working on a fix for it.

**Comment by jotadrilo** [comment]
This is fixed at #3299!
